kalite/distributed/management/commands/setup.py

```diff
@@ -250,11 +269,25 @@ def handle(self, *args, **options):
         call_command("syncdb", interactive=False, verbosity=options.get("verbosity"))
         call_command("migrate", merge=True, verbosity=options.get("verbosity"))
 
+        # download assessment items
         # This can take a long time and lead to Travis stalling. None of this is required for tests.
-        if not settings.RUNNING_IN_TRAVIS:
-            # download assessment items
-            # TODO-BLOCKER (aron): do not hardcode this, and put in the proper URL once we have a redirect set up
-            call_command("unpack_assessment_zip", "http://eslgenie.com/media/assessment_item_resources.zip")
+        if not settings.RUNNING_IN_TRAVIS and options['interactive']:
```

Comment:
Might be good to either default to downloading the assessment items in non-interactive mode, or at least have an additional command line argument that can force it to? Although I guess any automated deployment systems can also manually run `unpack_assessment_zip` afterwards as well...